wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -802,23 +813,12 @@ public final class SessionManager: NSObject, SessionManagerType {
         }
 
         delegate?.sessionManagerWillLogout(error: error, userSessionCanBeTornDown: { [weak self] in
-
-            if deleteCookie {
-                self?.environment.cookieStorage(for: account).deleteKeychainItems()
-            }
-
-            if deleteAccount {
-                activeUserSession.lastEventIDRepository.storeLastEventID(nil)
-            }
-
             let group = self?.dispatchGroup
             group?.enter()
 
-            activeUserSession.e2eiActivationDateRepository.removeE2EIActivationDate()
```

Comment:
I think it should be fine do not have it separate and remove all date in user defaults.